src/components/swap/SwapDetailsDropdown.tsx

```diff
@@ -11,15 +12,19 @@ import Row, { RowBetween, RowFixed } from 'components/Row'
 import { MouseoverTooltipContent } from 'components/Tooltip'
 import { SUPPORTED_GAS_ESTIMATE_CHAIN_IDS } from 'constants/chains'
 import { darken } from 'polished'
-import { useState } from 'react'
+import { useEffect, useState } from 'react'
 import { ChevronDown, Info } from 'react-feather'
 import { InterfaceTrade } from 'state/routing/types'
 import styled, { keyframes, useTheme } from 'styled-components/macro'
 import { HideSmall, ThemedText } from 'theme'
+import { computeRealizedLPFeePercent } from 'utils/prices'
 
 import { AdvancedSwapDetails } from './AdvancedSwapDetails'
+import { getPriceImpactPercent } from './AdvancedSwapDetails'
 import GasEstimateBadge from './GasEstimateBadge'
 import { ResponsiveTooltipContainer } from './styleds'
+import { formatPercentInBasisPointsNumber } from './SwapModalFooter'
+import { getNumberFormattedToDecimalPlace } from './SwapModalFooter'
```

Comment:
could you move these two functions to a utils file? maybe in the analytics folder? also you should be able to import in one line - VSCode has an "organize imports" setting which makes it nicer

Reply:
sounds good done